### ISSUE TITLE
Question about task accuracy on MoTIF

### ISSUE THREAD

**Issue by zerlinwang** [aburns4/MoTIF#6]
![image](https://user-images.githubusercontent.com/80957609/217462147-fa73e6ec-5d2d-4c07-8851-6fe498541ea3.png)

According to my understanding, Seq2Seq and MOCA were trained with datasets motif and rico_sca. Seq2Act was trained with rico_sca alone.  And all of them were tested on motif. Therefore, during testing phase, as shown in Table 4, all of apps from motif are ``App Unseen`` for Seq2Act (only trained with rico_sca). So what's the meaning of ``App Seen``?

If my understanding is wrong, please let me know.

**Comment by aburns4** [comment]
For the App Seen and App Unseen splits in Table 4, RicoSCA's apps are a superset of MoTIF's. So, all of the apps have been seen during training with the original RicoSCA data. For App Unseen, we removed the apps in RicoSCA training data that are in the MoTIF test splits. I meant to get jointly trained MoTIF + Rico results for Seq2Act and didn't manage to get in working in time for the deadline.

As for Table 14, I do not report App Seen / Task Unseen Curr or App Unseen / Task Seen for Seq2Act because it was not fine-tuned or trained jointly on MoTIF and would require this for tasks to have been seen in other apps than the test app. The ability to study task generalization across app environments is a unique quality of MoTIF. Table 14 in the supplementary just reports Seq2Seq and MOCA models.
